packages/xrpl/test/webpack.config.js

```diff
@@ -28,6 +28,7 @@ function webpackForTest(testFileName) {
       new webpack.DefinePlugin({
         'process.stdout': {},
       }),
+      new webpack.ProvidePlugin({ Buffer: ['buffer', 'Buffer'] }),
```

Comment:
Why was this changed?

Reply:
`Buffer` needs to be polyfilled in the browser, but given it's not used anymore, as Uint8Array is instead, it is removed. This was done to reduce the bundle size. 

Note that the library will still accept user Buffer objects (as they are subclasses of Uint8Array) 

```
> Buffer.of(0) instanceof Uint8Array
true
```

Reply:
Fixed by most recent commit https://github.com/XRPLF/xrpl.js/pull/2273/commits/5ff5e60d600b498da352fc5cd0c1ffbab5794a1f